### PR TITLE
Add blog-post for week 4 (2023)

### PR DESCRIPTION
**Pull request by majamassarini**

### CODE REVIEW

File: content/posts/weekly/January-2023.md [hunk 1]
```diff
@@ -31,3 +31,9 @@ weight: 54
...
 - We have fixed a bug which caused long Copr build end reporting time on a few occurrences. ([packit-service#1838](https://github.com/packit/packit-service/pull/1838))
 - A few fixes has been implemented to make it possible to use [`propose-downstream` job](https://packit.dev/docs/configuration/#propose_downstream) on GitLab. ([packit-service#1842](https://github.com/packit/packit-service/pull/1842), [packit-service#1844](https://github.com/packit/packit-service/pull/1844), [packit-service#1845](https://github.com/packit/packit-service/pull/1845), [packit-service#1846](https://github.com/packit/packit-service/pull/1846))
+
+## Week 3 (January 17th – January 23rd)
+
+- Users can now re-trigger a _Bodhi update_ or a _Koji build_ writing comments like `/packit create-update` and `/packit koji-update` in the issue opened by Packit in the issues repository if anything went wrong during the _propose-downstream_ ([packit-service#1796](https://github.com/packit/packit-service/pull/1796))
+- Update React to v18 and related packages ([dashboard#203](https://github.com/packit/dashboard/pull/203))
```
Comment (nforro) [inline content/posts/weekly/January-2023.md:38]:
```suggestion
- React and related packages were updated to v18  ([dashboard#203](https://github.com/packit/dashboard/pull/203))
```

File: content/posts/weekly/January-2023.md [hunk 2]
```diff
@@ -31,3 +31,9 @@ weight: 54
...
 - We have fixed a bug which caused long Copr build end reporting time on a few occurrences. ([packit-service#1838](https://github.com/packit/packit-service/pull/1838))
 - A few fixes has been implemented to make it possible to use [`propose-downstream` job](https://packit.dev/docs/configuration/#propose_downstream) on GitLab. ([packit-service#1842](https://github.com/packit/packit-service/pull/1842), [packit-service#1844](https://github.com/packit/packit-service/pull/1844), [packit-service#1845](https://github.com/packit/packit-service/pull/1845), [packit-service#1846](https://github.com/packit/packit-service/pull/1846))
+
+## Week 3 (January 17th – January 23rd)
+
+- Users can now re-trigger a _Bodhi update_ or a _Koji build_ writing comments like `/packit create-update` and `/packit koji-update` in the issue opened by Packit in the issues repository if anything went wrong during the _propose-downstream_ ([packit-service#1796](https://github.com/packit/packit-service/pull/1796))
```
Comment (nforro) [inline content/posts/weekly/January-2023.md:37]:
```suggestion
- Users can now re-trigger a _Bodhi update_ or a _Koji build_ by writing comments like `/packit create-update` and `/packit koji-update` in an issue opened by Packit in the issues repository if anything went wrong during the _propose-downstream_ job ([packit-service#1796](https://github.com/packit/packit-service/pull/1796))
```

File: content/posts/weekly/January-2023.md (hunk 1 above)
Comment (jpopelka) [reply content/posts/weekly/January-2023.md:38]: sorry to ruin the party, but I don't think users are interested in such info

**Review by jpopelka — COMMENTED** [review]
I'm missing release notes from packit/packit#1827

**Comment by nforro** [comment]
> I'm missing release notes from [packit/packit#1827](https://github.com/packit/packit/pull/1827)

And also from https://github.com/packit/packit/pull/1829 and https://github.com/packit/packit/pull/1825, the same PRs that were skipped when generating release changelog on Friday. It looks like release notes are missing from merge commits for some reason.

**Comment by majamassarini** [comment]
Thank you both! I hope I fixed all. However I am not going to merge this since we reverted images in  production.

**Review by lachmanfrantisek — COMMENTED** [review]
Looks like this was included in #599 so we can probably close this pull-request...

**Pull request closed without merging**